Server Side Import

I am trying to import this on the serverside and am getting the following error.

```import { Html } from "@react-email/html";```

```import { Html } from "@react-email/html";
         ^^^^
SyntaxError: The requested module '@react-email/html' does not provide an export named 'Html'
    at ModuleJob._instantiate (internal/modules/esm/module_job.js:92:21)
    at async ModuleJob.run (internal/modules/esm/module_job.js:107:20)
    at async Loader.import (internal/modules/esm/loader.js:179:24)
```

Note: I discovered later that the exports in the package.json file had broken the `email dev` server, giving the error message "Module not found: Default condition should be last one" pointing to the line where components were being imported from @react-email/components.

What caused it was that I had the "types" and "default" conditions in the json snippet above the wrong way around.

I've fixed the snippet above so that the default value being specified for the import and require sections is the last condition so if anyone else comes across the thread they don't copy/paste the wrong information, and the `email dev` server is working again - but I wanted to highlight that it's important that default lines come last in their sections otherwise the nextjs server will be unable to load the email templates.

--- a/packages/components/package.json
+++ b/packages/components/package.json
@@ -9,6 +9,18 @@
   "files": [
     "dist/**"
   ],
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.mjs"
+      },
+      "require": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.js"
+      }
+    }
+  },
   "license": "MIT",
   "scripts": {
     "build": "tsup src/index.ts --format esm,cjs --dts --external react",

